libraries/CORE_NAILS_Form_validation.php

```diff
@@ -90,6 +90,40 @@ public function valid_postcode($str)
 
     // --------------------------------------------------------------------------
 
+    /**
+     * Checks if an array satisfies specified count restrictions.
+     * @param  array  $aArray  The value to check
+     * @param  string  $sParam  The parameter to check against
+     * @return boolean
+    */
+    public function item_count(array $aArray, $sParam) {
```

Comment:
Not sure what @hellopablo wanted to do but I think camelCase for method names was preferred! I may be wrong though!